wpengine/faustjs
# [Merl-1281] Add extra Google Analytics variables to Faust

## Tasks


## Description

Adds the following variables to GA:

**node_faustwp_block_editor_utils_version**: Tracks block-editor-utils version.
**node_faustwp_experimental_app_router_version**: Tracks experimental app router version.

## Related Issue(s):



## Testing

1. Build cli package with the updates.
2. Set FAUST_DEBUG: true env variable
3. Ensure telemetry is enabled
4. Run npm `run dev -w examples/next/block-support` and `npm run dev -w examples/next/app-router` and observer the telemetry event payload. It should contain the relevant package versions.

```bash
debug - Telemetry event:  {
  node_faustwp_core_version: '1.2.0',
  node_faustwp_cli_version: '1.2.1',
  node_faustwp_blocks_version: '2.0.0',
  node_faustwp_block_editor_utils_version: '0.0.5',
  node_faustwp_experimental_app_router_version: undefined,
  node_apollo_client_version: '3.8.8',
```


## Screenshots



## Documentation Changes



## Dependant PRs
